packages/domain/lib/src/mappers/duration_to_string.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:domain/src/mappers/base_mapper.dart';
+
+class DurationToStringMapper extends Mapper<int?, String> {
+  @override
+  String call(int? params) {
+    if (params == null) return '';
+    final int hours = params ~/ 60;
+    final minutes = params - hours * 60;
+    if (hours == 0) return ' ${minutes}m';
+    return ' ${hours}h ${minutes}m';
+  }
```

Comment:
optional, I prefer to pass non-nullable params to the mapper. we can do it in the following way:

class DurationMapper extends Mapper<int, String> {
String call(int params) {}
}

value != null ? durationMapper(value) : "";

justification - handling null values is not mapper's responsibility